ui/modal/info_modal.go

```diff
@@ -45,22 +47,33 @@ type InfoModal struct {
 }
 
 func NewInfoModal(l *load.Load) *InfoModal {
-	return NewInfoModalWithKey(l, "info_modal")
+	return NewInfoModalWithKey(l, "info_modal", false)
 }
 
-func NewInfoModalWithKey(l *load.Load, key string) *InfoModal {
+// This function for normal positive button
+func NewInfoModal2(l *load.Load) *InfoModal {
+	return NewInfoModalWithKey(l, "info_modal", true)
+}
+
+func NewInfoModalWithKey(l *load.Load, key string, isPositiveButtonNormal bool) *InfoModal {
 
 	in := &InfoModal{
-		Load:         l,
-		Modal:        l.Theme.ModalFloatTitle(key),
-		btnPositve:   l.Theme.OutlineButton(values.String(values.StrYes)),
-		btnNegative:  l.Theme.OutlineButton(values.String(values.StrNo)),
-		isCancelable: true,
-		isLoading:    false,
-		btnAlignment: layout.E,
+		Load:             l,
+		Modal:            l.Theme.ModalFloatTitle(key),
+		btnNegative:      l.Theme.OutlineButton(values.String(values.StrNo)),
+		isCancelable:     true,
+		isLoading:        false,
+		btnAlignment:     layout.E,
+		btnPositiveWidth: 0,
+	}
+
+	if isPositiveButtonNormal {
+		in.btnPositive = l.Theme.Button(values.String(values.StrYes))
+	} else {
+		in.btnPositive = l.Theme.OutlineButton(values.String(values.StrYes))
```

Comment:
You don't need this. The outline button is the regular button with out its background. You can practically just use `PositiveButtonStyle()` to style the button to what you want it to look like.

Reply:
@Sirmorrison I understand what you said but PositiveButtonStyle fuction should be used with undefined buttons in the application, and for defined buttons should use those buttons instead of having to custom a defined button.